packages/keystone/asyncLocalStorage.js

```diff
@@ -0,0 +1,27 @@
+const { AsyncLocalStorage } = require('node:async_hooks')
+
+const { getLogger } = require('./logging')
+
+const ASYNC_LOCAL_STORAGES = {}
```

Comment:
Why do you need multiple copies of AsyncLocalStorage? You can't store all of your data inside single instance?

Reply:
You can technically store everything in single ALS, but I believe it would be better to split them to different objects